src/sentry/search/utils.py

```diff
@@ -18,6 +18,15 @@ class InvalidQuery(Exception):
     pass
 
 
+def parse_release(project, value):
```

Comment:
i was gonna add environment support, but its pretty tricky to query that at the moment, and id rather when we do it we just precompute it